index.md

```diff
@@ -280,15 +285,14 @@ Information on web accessibility:
   **screen readers** that read aloud web pages for people who cannot read text, **screen magnifiers** for people with some types of low vision, and **voice recognition software** and **selection switches** for people who cannot use a keyboard or mouse.
 
 <span id="ald">ALDs, hearing loops, induction loops</span>
-:  Assistive listening devices (ALDs), hearing loops, and induction loops amplify sound from the speaker's microphone.
+:  Assistive listening devices (ALDs), hearing loops, and induction loops amplify sound from the speaker's microphone for people who are hard of hearing.
 
-<span id="cart">CART</span>
-:  CART, Computer Aided Real–Time Captioning or Communication Access Realtime Translation, is when a professional types what is being  said verbatim so that people can read the text output.
+<span id="captions">captions, captioner</span>
+:  _Captions_ (also called “subtitles”) are a text version of speech and important non-speech audio. Live captioning is also called CART, Computer Aided Real–Time Captioning or Communication Access Realtime Translation.<br>A _captioner_ is a professional who provides what is being said verbatim so that people can read the text output.<br>More information is in [Captions/Subtitles, in Making Audio and Video Media Accessible](/media/av/captions/).
```

Comment:
Not sure about "also called subtitles". Maybe "in some regions and languages called subtitles" instead? It is longer but clearer that captions are not also called subtitles but only sometimes referred to as such in specific situations.

Reply:
"Captions (sometimes called "subtitles")"?

Would that be a bit shorter?

Reply:
Well, not in English - that's the issue IMO

Reply:
(called “subtitles” in some areas)

Reply:
+1 to @slhenry

Reply:
+1 to @slhenry